### PR TITLE
fix(TeamsCard):Added Member Profile Picture

### PR DESCRIPTION
**Pull request by sanjaykrishna1203**
### Functionality:

- Fixed Duplicate Teams Issue.
-Replaced Members' profile picture instead of Emails

### Solution:

- Fetched the Team Details Instead of checking the Existing Team Detail.
- Created a profile-picture component

### Risk level:
- [x] high 
- [ ] medium
- [ ] low

### How to test:

- Go to My organization check whether there are duplicate teams.
- Add Members and check whether the profile pictures are displayed.

### CODE REVIEW

File: src/app/body/common-tools/profile-pictures/profile-pictures.component.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+* 
+* This program is distributed in the hope that it will be useful, 
+* but WITHOUT ANY WARRANTY; without even the implied warranty of 
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
+* See the MIT License for more details. 
+***********************************************************/-->
+<ng-container *ngIf="photoUrlReady">
+    <ng-container *ngFor="let item of ProfileList">
+        <span class="avatar">
+            <img [src]='item' onerror= "this.src = '../../assets/defaultavatar.jpg'" width="40" height="40"/>
```
Comment (Pendyala-thanvi) [inline src/app/body/common-tools/profile-pictures/profile-pictures.component.html:18]: Is this path for the default avatar correct?

Comment (sanjaykrishna1203) [reply src/app/body/common-tools/profile-pictures/profile-pictures.component.html:18]: Yes, I took it from the suggestion bucket.

Comment (Pendyala-thanvi) [reply src/app/body/common-tools/profile-pictures/profile-pictures.component.html:18]: The path is wrong there too. Please fix it there as well.

File: src/app/services/applicationSettings/application-settings.service.ts [hunk 2]
```diff
@@ -52,7 +52,6 @@ export class ApplicationSettingsService {
   constructor(private userService: UserServiceService, private backendService: BackendService, private functions: AngularFireFunctions, private authService: AuthService) { }
 
   getTeamDetails(teamId: string) {
-    if(this.team == undefined || this.team.TeamId != teamId) {
```
Comment (viv9k) [inline src/app/services/applicationSettings/application-settings.service.ts:55]: Why are we doing these changes

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do the changes

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**